data/json/recipes/armor/storage.json

```diff
@@ -305,6 +305,20 @@
     "qualities": [ { "id": "SEW", "level": 1 } ],
     "components": [ [ [ "leather", 6 ], [ "tanned_hide", 1 ] ], [ [ "filament", 300, "LIST" ], [ "cordage", 1, "LIST" ] ] ]
   },
+  {
+    "result": "stone_pouch",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "throw", 1 ] ],
+    "time": "1 h 20 m",
+    "autolearn": true,
+    "qualities": [ { "id": "SEW", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ [ [ "rag", 8 ] ], [ [ "plastic_chunk", 4 ] ], [ [ "filament", 600, "LIST" ], [ "cordage", 2, "LIST" ] ] ]
```

Comment:
You may want to use leather instead of plastic for this; while it's possible to acquire plastic by finding trash in forest, it probably shouldn't be necessary to make an item that is mainly useful for wilderness situations.

Reply:
good idea